scale/job/serializers.py

```diff
@@ -281,25 +298,41 @@ class JobExecutionSerializer(JobExecutionBaseSerializer):
     error = ErrorBaseSerializer(source='jobexecutionend.error')
 
 
-class JobExecutionDetailsSerializer(JobExecutionSerializer):
+class JobExecutionSerializer(JobExecutionBaseSerializer):
+    """Converts job execution model fields to REST output"""
+
+    from error.serializers import ErrorBaseSerializer
+    from node.serializers import NodeBaseSerializer
+
+    job = ModelIdSerializer()
+    node = NodeBaseSerializer()
+    error = ErrorBaseSerializer(source='jobexecutionend.error')
+    job_type = JobTypeBaseSerializer()
+
+    exe_num = serializers.IntegerField()
+    cluster_id = serializers.CharField()
+    input_file_size = serializers.FloatField()
+
+
+# TODO: remove this function when REST API v5 is removed
+class OldJobExecutionDetailsSerializer(OldJobExecutionSerializer):
     """Converts job execution model fields to REST output"""
+
     from error.serializers import ErrorSerializer
     from node.serializers import NodeSerializerV4
 
     job = JobSerializer()
     node = NodeSerializerV4()
     error = ErrorSerializer(source='jobexecutionend.error')
 
-    environment = serializers.JSONField(default=dict)
-    cpus_scheduled = serializers.FloatField()
-    mem_scheduled = serializers.FloatField()
-    disk_in_scheduled = serializers.FloatField(source='input_file_size')
-    disk_out_scheduled = serializers.FloatField()
-    disk_total_scheduled = serializers.FloatField()
 
-    results = serializers.JSONField(default=dict, source='jobexecutionoutput.output')
+class JobExecutionDetailsSerializer(JobExecutionSerializer):
+    """Converts job execution model fields to REST output"""
 
-    results_manifest = serializers.JSONField(default=dict)
+    task_results = serializers.JSONField(default=dict)
```

Comment:
task_results will need its source to point to the JobExecutionEnd model.

Also I think it would be better if the JSON fields default to null (like error model above) instead of being an empty dict.